cockroachcloud/tools-page.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Monitoring Page
-summary: How to use the Monitoring page to enable monitoring with Datadog and access the DB Console.
+title: Tools Page
+summary: How to use the Tools page to enable monitoring with Datadog and access the DB Console.
 toc: true
 docs_area: manage
 ---
 
-The **Monitoring** page is accessible on {{ site.data.products.dedicated }} clusters. This page allows you to:
+The **Tools** page is accessible on {{ site.data.products.dedicated }} clusters. This page allows you to:
```

Comment:
It's also only available for clusters running v22.2 or later. I think we should call that out but I don't know if we have a precedent of doing that in the Cloud docs... I'll look around. What do you think?